projects/components/src/titled-content/titled-content.component.ts

```diff
@@ -75,3 +99,18 @@ export const enum TitlePosition {
   Header = 'header',
   Footer = 'footer'
 }
+
+export const enum TitleStyle {
+  Regular = 'regular',
+  GrayedOut = 'grayed-out'
+}
+
+export const enum LinkButtonStyle {
```

Comment:
prefix all these enums with `TitledContent` since these are all specific to this component.

Reply:
already frefixed